### PR TITLE
Set dynamic height with fadeIn true

### PR DESCRIPTION
**Pull request by andrelmlins**
Hello @moaazsidat and @renatobenks , great library, congratulations.

I cannot set the camera height when using fadeIn. What do you think of this solution?

### CODE REVIEW

**Comment by andrelmlins** [comment]
Hi there, any news?

File: index.js [hunk 1]
```diff
@@ -271,15 +271,20 @@ export default class QRCodeScanner extends Component {
...
+      }
 
 			if (this.props.fadeIn) {
 				return (
 					<Animated.View
 						style={{
 							opacity: this.state.fadeInOpacity,
 							backgroundColor: 'transparent',
-							height: styles.camera.height,
+							height,
```
Comment (renatobenks) [inline index.js:287]: try to consider a simpler solution for this problem like this one:

https://github.com/jkrecek/react-native-qrcode-scanner/blob/27e38e17f023e69b403c4141b0a3d84a6a6e044f/index.js#L282

**Review by renatobenks — COMMENTED** [review]
thanks for the contribution, it means a lot!

**Review by renatobenks — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (andrelmlins) [reply index.js]: done, when will a new version be created?

**Comment by andrelmlins** [comment]
Thanks @renatobenks, when will a new version be created?

**Comment by renatobenks** [comment]
that's a hard one @andrelmlins, because the process to release new versions at npm is not automated, and I need of @moaazsidat to publish it, and he's not so available to do that lately.

But what I'm trying to move forward with is to automate that process to avoid this dependency for good

**Comment by andrelmlins** [comment]
Ok @renatobanks, thanks very much, if you need help i can do a PR for automation scripts.

**Pull request merged**